AppWidget/glance-widget/src/main/java/com/example/android/glancewidget/FirstGlanceWidget.kt

```diff
@@ -47,26 +48,30 @@ class FirstGlanceWidget : GlanceAppWidget() {
         Column(
             modifier = GlanceModifier
                 .fillMaxSize()
-                .background(day = Color.White, night = Color.LightGray)
+                .background(day = Color.White, night = Color.Black)
                 .appWidgetBackground()
                 .cornerRadius(16.dp)
```

Comment:
The previous is using the system's radius ... we should do the same here then. For this, you need to define a dimension resource, which resolves to the system one on Android S+ and to 16dp on Android R- and use it here.

Reply:
Done.